src/components/ResourceList/components/FilterControl/components/DateSelector/tests/DateSelector.test.tsx

```diff
@@ -161,6 +161,35 @@ describe('<DateSelector />', () => {
     });
   });
 
+  describe('timezones adjustments', () => {
+    const date = '2019-01-01';
+    let getTimezoneOffset: jest.SpyInstance;
+
+    beforeEach(() => {
+      getTimezoneOffset = jest.spyOn(Date.prototype, 'getTimezoneOffset');
```

Comment:
Nice! If you are only mocking it for one test, it can be done in the test itself.